File: Day 2/1_project/app/src/main/java/com/example/project_1/MainActivity.java
package com.example.project_1;

import androidx.appcompat.app.AppCompatActivity;
import android.view.View;

import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {


    public void sendNow(View view)
    {
        Toast.makeText(this, "Data is sending....", Toast.LENGTH_SHORT).show();
    }

    public void receivedNow(View view)
    {
        Toast.makeText(this, "Data is Received....", Toast.LENGTH_SHORT).show();
    }

    public void deleteNow(View view)
    {
        Toast.makeText(this, "Data is Deleting....", Toast.LENGTH_SHORT).show();
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }




}

File: Siddhi mam Tasks/Day 42/OptionsMenu/app/src/main/java/com/example/optionsmenu/MainActivity.kt
package com.example.optionsmenu

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu

import android.view.MenuItem
import android.widget.Toast
import androidx.appcompat.widget.Toolbar


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Find the toolbar in the layout and set it as the ActionBar
        val toolbar: Toolbar = findViewById(R.id.toolbar)
        toolbar.title = "Select the Options"
        setSupportActionBar(toolbar)
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        // Inflate the menu; this adds items to the action bar if it is present.
         menuInflater.inflate(R.menu.menu_main, menu)
         return true
    }



    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        return when (item.itemId) {
            R.id.itFacebook -> {
                Toast.makeText(applicationContext, "click on facebook", Toast.LENGTH_LONG).show()
                true
            }
            R.id.itTwitter ->{
                Toast.makeText(applicationContext, "click on twitter", Toast.LENGTH_LONG).show()
                return true
            }
            R.id.itSnapchat ->{
                Toast.makeText(applicationContext, "click on snapchat", Toast.LENGTH_LONG).show()
                return true
            }
            R.id.itInstagram ->{
                Toast.makeText(applicationContext, "click on instagram", Toast.LENGTH_LONG).show()
                return true
            }
            else -> super.onOptionsItemSelected(item)
        }
    }
}